### PR TITLE
Astro Runtime Versioning and Lifecycle Policy Topic Reorganization

### LINKED ISSUE #892: Reorganize Astro Runtime Versioning and Lifecycle Policy
@paolaperaza has requested the following organizational structure in runtime-version-lifecycle-policy.md:

- Release Channels
- Astro runtime Maintenance Policy
- Backport policy for bug and security fixes
- Astro Runtime lifecycle schedule

Also, "rename this doc to just Astro Runtime lifecycle policy without the “versioning” since that’s no longer in there".

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #892

### CODE REVIEW

File: astro/runtime-version-lifecycle-policy.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Astro Runtime versioning and lifecycle policy"
-sidebar_label: "Astro Runtime versioning and lifecycle policy"
+title: "Astro Runtime lifecycle policy"
```
Comment (jwitz) [inline astro/runtime-version-lifecycle-policy.md:2]:
```suggestion
title: "Astro Runtime maintenance and lifecycle policy"
```
@paolaperaza thoughts? Or is maintenance an inherent detail of lifecycles in this context?

**Review by paolaperaza — APPROVED** [review]
Looks good @kwcanuck - thank you! Minor comments in here but will let you ✅

File: astro/runtime-version-lifecycle-policy.md [hunk 2]
```diff
@@ -9,21 +9,18 @@ Astro Runtime is a production ready, data orchestration tool based on Apache Air
```
Comment (paolaperaza) [inline astro/runtime-version-lifecycle-policy.md:9]: Could link to arch doc at end of this?

File: astro/runtime-version-lifecycle-policy.md (hunk 1 above)
Comment (jwitz) [reply astro/runtime-version-lifecycle-policy.md:2]: I think "Astro Runtime" has a nice ring to it and helps differentiate from uncapitalized uses of runtime. Maybe we can revisit this once Certified is out of the picture.

**Pull request merged**